trackpy/utils.py

```diff
@@ -288,3 +289,21 @@ def _pandas_sort_post_017(df, by, *args, **kwargs):
     pandas_sort = _pandas_sort_post_017
 else:
     pandas_sort = _pandas_sort_pre_017
+
+@contextmanager
```

Comment:
These are nice. You should consider contributing these upstream to pandas, @caspervdw.